src/Sql/Sql.Test/ScenarioTests/ManagedInstanceCrudScenarioTests.ps1

```diff
@@ -68,11 +68,12 @@ function Test-CreateManagedInstance
 		$edition = "GeneralPurpose"
 		$computeGeneration = "Gen4"
 		$managedInstanceName = Get-ManagedInstanceName
+		$dnsZonePartner = $managedInstance1.ResourceId
```

Comment:
You should verify that this value comes back, and re-record the test for the new value

Reply:
for some reason the test is passing locally with the new record session with playback mode but not in the check-in automation.